src/blocks/food-and-drinks/food-item/edit.js

```diff
@@ -227,6 +227,26 @@ class FoodAndDrinksEdit extends Component {
 								{ ...richTextAttributes }
 							/>
 							<div className="wp-block-coblocks-food-item__attributes">
+								{ isSelected && attributes.title ? (
+									<span>
+										<IconButton
+											icon={ icons.popular }
+											className="wp-block-coblocks-food-item__attribute wp-block-coblocks-food-item__attribute--popular"
+											onClick={ () =>
+												setAttributes( { popular: ! attributes.popular } )
+											}
+											label={ __( 'Popular' ) }
+											isToggled={ attributes.popular }
+										/>
+									</span>
```

Comment:
https://github.com/godaddy/coblocks/pull/958/files#diff-a666a2948d2f4865c59523949605e3d2R231-R241
Let's go ahead and just show the toggle in the sidebar and remove this.

Reply:
Actually, I think it'd be better to have `popular` be the first item in the attribute shortcuts, dropping Gluten Free from these shortcuts.

Reply:
Adding my notes from Slack here for future reference, then resolving this request.

> Alternatively, should we instead make a style available to the menu-item block? I don’t think the icon really fits well with the category of icons there. Maybe a block style for a popular item (or special, or featured)?

> The existing icons describe the actual food item. At a quick glance someone could see that an item is spicy and either consciously gravitate towards those items or avoid them.

> The star for popular doesn’t feel like it has that same “filter”. I’m just wondering what type of food a star is.

> A style for a popular/special/featured item would draw the attention because it’s styled differently whereas the attribute icons would not but rather provide a mental filter. Similar to how the food items with huge pictures on a menu draw your gaze away from the general list of food items.

> I'm not suggesting we do a completely different layout for a popular item. Instead, applying minor styles to emphasize an item similar to how we have different styles for the alert block: same layout but different ways of communicating.